bitbots_vision/cfg/Vision.cfg

```diff
@@ -40,8 +40,8 @@ group_white_color_detector = group_color_detector.add_group("white_color_detecto
 group_red_color_detector = group_color_detector.add_group("red_color_detector")
 group_blue_color_detector = group_color_detector.add_group("blue_color_detector")
 
+group_ball_fcnn.add("ball_fcnn_publish_debug_img", bool_t, 0, "toggles publishing of the fcnn heatmap image for debug purposes", None)
```

Comment:
This parameter is added twice (group_ball_fcnn and group_ROS). I would prefer to put it in group_ROS only since the debug image setting is also set there